### ISSUE TITLE
Bug: `patch` does not remove `className` attribute from element

### ISSUE THREAD

**Issue by tkofh** [aidenybai/million#153]
### Describe the bug

```markdown
When calling `patch` to swap a VNode with a className to one without a className, the className is not removed from the actual element.
```

### To Reproduce

[Codesandbox Reproduction](https://codesandbox.io/s/million-no-remove-class-attribute-fcgq5?file=/src/index.ts)

```typescript
import { m, createElement, patch } from "million";

const withClass = m("div", { className: "blue-text" }, ["Hello"]);

const element = createElement(withClass);

document.body.appendChild(element);

const withoutClass = m("div", {}, ["World"]);

patch(element, withoutClass);

const withEmptyClass = m("div", { className: "" }, ["World"]);

patch(element, withEmptyClass);
```

At the end of this script, after a `patch` with no className prop and one with an empty string, `blue-text` will still be attached to the HTML element.

### Expected behavior

```markdown
At the end of the above script, the HTML would not have a `blue-text` class in it.
```

### Operating System

MacOS Monterey

### Browser

Google Chrome 90

**Comment by aidenybai** [comment]
Fixed in #154

**Comment by aidenybai** [comment]
Patch in v0.7.1
